access_point/server/README.md

```diff
@@ -16,14 +16,16 @@ This server can be access on port 5000 when on the same network and `http:10.0.0
  * 'wifissid' is the SSID of the WIFI network you want to configure
  * 'wifipassd' is the PASSWORD of that WIFI network
 
-* /config/'stt'/'tts'/'hotword'/'wake'/'auth'/'email'/'passwd'
+* /config/'stt'/'tts'/'hotword'/'wake'
```

Comment:
make something like this

* /config/'stt'/'tts'/'hotword'/'wake'
  - 'stt' is the name of Speech To Text Service Provider
       - You can choose from 'google', 'ibm', 'bing', 'sphinx'
  - 'tts' is the name of Text To Speech Provider
       - You can choose from 'google', 'ibm', 'flite'
   -  'hotword' is the choice if you want to use Snowboy as the service for hotword prediction
       - You can choose from 'y' or 'n''
* /auth/'choice'/'email'/'password'  
   - choice' is the choice if you want to use the speaker in the authenticated mode
        -  You can choose from 'y' or 'n'
   - 'email' is the email of the user
   - 'passwd' is the corresponding password